README.md

```diff
@@ -1,26 +1,9 @@
 # Web3Signer documentation
 
-[Web3Signer] is a transaction signing application to be used with a web3 provider. The software sources are hosted in the [Web3Signer] repository.
+Web3Signer is a transaction signing application to be used with a web3 provider.
+The software sources are hosted in the [Web3Signer repository](https://github.com/Consensys/web3signer).
 
-This documentation repo is built using [Docusaurus 2](https://docusaurus.io/).
+This documentation repository is built using [Docusaurus](https://docusaurus.io/), and the doc
+site is published at [`docs.web3signer.consensys.net`](https://docs.web3signer.consensys.net/).
 
-See [more](https://docs-template.consensys.net/) information about using Docusaurus quickly.
-
-### Contributing & Local Development
-
-    $ npm install
-    $ npm run prepare
-    $ npm start
-
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
-
-### Build
-
-    $ npm run build
-    $ npm run serve
-
-This command generates static content into the `build` directory.
-
-<!--- Links -->
-
-[Web3Signer]: https://github.com/Consensys/web3signer
+View the [ConsenSys doc contribution guidelines](https://docs-template.consensys.net/).
```

Comment:
Perhaps mention that they can view the guidelines for instructions to publish locally and view styling info?